src/PrestaShopBundle/Controller/Admin/CombinationController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PrestaShopBundle\Controller\Admin;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use PrestaShop\PrestaShop\Adapter\CombinationDataProvider;
+use PrestaShopBundle\Form\Admin\Product\ProductCombination;
+
+class CombinationController extends Controller
+{
+    public function generateCombinationFormAction($combinationIds)
+    {
+        $response = new Response();
+        $result = '';
+
+        $combinations = explode('-', $combinationIds);
+        if ($combinationIds == 0 || count($combinations) == 0) {
+            return $response;
+        }
+
+        $combinationDataProvider = new combinationDataProvider();
+
+        foreach ($combinations as $combinationId) {
+            $form = $this->createForm(ProductCombination::class, $combinationDataProvider->getFormCombination($combinationId));
```

Comment:
Other solution, embed your form using a ``CollectionType`` but except if I'm wrong, you won't have any control on the generated indexes.